ext/odbc/tests/odbc_field_num_001.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+odbc_field_scale(): Getting the scale of the field
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+
+odbc_exec($conn, 'CREATE TABLE field_num (foo INT, bar REAL, baz VARBINARY(50))');
+odbc_exec($conn, "INSERT INTO field_num VALUES (1, 3.14, CONVERT(VARBINARY(50), 'baz'))");
+
+$res = odbc_exec($conn, 'SELECT * FROM field_num');
+var_dump(odbc_field_num($res, "foo"));
+var_dump(odbc_field_num($res, "bar"));
+var_dump(odbc_field_num($res, "baz"));
+var_dump(odbc_field_num($res, "nonexistent"));
```

Comment:
No warning huuum, which I suppose is fine as that's probably the only reason it can return false

Reply:
Yeah, that's the case!